en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageByGeoLocation
+
+Represents a single API usage across different countries.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usageByGeoLocation</strong></td>
+<td valign="top">[<a href="#usagebygeolocation">UsageByGeoLocation</a>]</td>
+<td>
+
+List containing usages of API in different countries. Returns an empty array if no data available.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsageOverTime
+
+Represents a single API usage over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#apiusage">APIUsage</a>]</td>
+<td>
+
+List containing the API usage values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### AlertSummaries
+
+Represents alert summaries.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>usage</strong></td>
+<td valign="top">[<a href="#alertsummary">AlertSummary</a>]</td>
+<td>
+
+List containing alert summary for each alert. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>pagination</strong></td>
+<td valign="top"><a href="#pagination">Pagination</a>!</td>
+<td>
+
+Pagination details.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### AlertSummary
+
+Represents a summary for an alert.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>timestamp</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
```

Comment:
When you say "used granularity value" it is not clear as to where it is used and by what. Please check whether the following suggestion gives the same meaning.

```suggestion
Represents a timestamp value that is calculated based on the specified granularity value.  for example, if the granularity value is `1d`, the timestamp represents the start of the day calculated adhering to the timezone provided in the time filter.
e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
```